pkg/docker/docker.go

```diff
@@ -472,17 +480,50 @@ func (d *stiDocker) RunContainer(opts RunContainerOptions) (err error) {
 		}
 	}
 
-	glog.V(2).Infof("Waiting for container")
-	exitCode, err := d.client.WaitContainer(container.ID)
-	wg.Wait()
-	if err != nil {
-		return err
-	}
-	glog.V(2).Infof("Container exited")
+	if opts.TargetImage {
 
-	if exitCode != 0 {
-		return errors.NewContainerError(container.Name, exitCode, "")
+		lco := docker.ListContainersOptions{
+			All: true,
+		}
+		conts, lcerr := d.client.ListContainers(lco)
+		var liveports []docker.APIPort
+		if lcerr == nil {
+			for _, cont := range conts {
+				if cont.ID == container.ID {
+					liveports = cont.Ports
+					break
+				}
+			}
+		}
+
+		glog.V(0).Infof("\n\n\n\n\nThe image %s has been started in container %s as a result of the --run=true option.  The container's stdout/stderr will be redirected to this command's glog output to help you validate its behavior.  You can also inspect the container with docker commands if you like.  If the container is set up to stay running, you will have to Ctrl-C to exit this command, which should also stop the container %s.  This particular invocation attempts to run with the port mappings %+v \n\n\n\n\n", image, container.ID, container.ID, liveports)
```

Comment:
Just checked it and I have to complain :disappointed: because the ports are NOT mapped. I can't access localhost:8080 after doing: 
```
sti build https://github.com/openshift/sti-python.git --run --loglevel=3 --context-dir=3.4/test/setup-test-app/ openshift/python-34-centos7 python-sample-app
```
Running `docker run -p 8080:8080 python-sample-app` allows me to access the ports.